### ISSUE TITLE
Problem with Linux legacy addons when their folder isn't lower-case

### ISSUE THREAD

**Issue by thegrb93** [Facepunch/garrysmod-issues#4527]
People using Linux report Starfall not loading because the folder isn't lower-cased. When they change StarfallEx to starfallex, it works. I don't use Linux so can't test for myself, but seems like a gmod bug. https://github.com/thegrb93/StarfallEx/issues/911

**Comment by Kefta** [comment]
This has been a "bug" since GMod's inception, I assume it was intended by this point.

**Comment by robotboy655** [comment]
This is the only solution to addons developed on Windows, which has case-insensitive file system to also work under Linux/OSX, both of which have case-sensitive file systems. This is very important for Steam Workshop, where all files in the .gma are forced lowercase.

There are no plans in changing this behavior.

**Comment by thegrb93** [comment]
Why not force the legacy addon's filename lower-case? I don't understand why the user has to do it manually.

**Comment by Kefta** [comment]
"The first one" is not defined, and will just lead to more conflicts. I think GMod should instead spit out a console error when it ignores a file with uppercase, maybe in ``developer``.

**Comment by robotboy655** [comment]
Yeah I agree, it should tell the user that an added was refused to be mounted.

**Comment by thegrb93** [comment]
It would be most convenient if it worked in the absence of a conflict though, and emitted an error otherwise.

**Comment by DBotThePony** [comment]
Gmod actually follow source engine's behavior on filenames case-sensivity (source engine lowercase all paths on both platforms when it tries to access assets). For *dirty* workaround you might want to create a JFS disk partition with -O flag.
